configs/qcom_sdm636_bbry_luna_defconfig

```diff
@@ -0,0 +1,15 @@
+# Configuration for building U-Boot as 2nd stage
+# bootloader to be flashed as part of boot.img to
+# boot partition on Blackberry KEY2 LE phones.
+
+#include "qcom_sdm660_defconfig"
+
+# luna device tree is not upstream yet
+CONFIG_OF_UPSTREAM=n
+CONFIG_DEFAULT_DEVICE_TREE="qcom/sdm636-bbry-luna"
+
+CONFIG_REQUIRE_SERIAL_CONSOLE=n
+CONFIG_SERIAL_PRESENT=n
+
```

Comment:
what about adding `CONFIG_NO_FB_CLEAR=n` as it was done in #21 , if picture from stock bootloader interferes with u-boot output?

Reply:
added